webpack/assets/javascripts/foreman_dashboard.js

```diff
@@ -5,7 +5,7 @@ import { activateTooltips } from './foreman_tools';
 import { translate as __ } from './react_app/common/I18n';
 
 $(document).on('ContentLoad', () => {
-  if (window.foreman_url('/') === window.location.pathname) {
+  if (tfm.tools.foremanUrl('/') === window.location.pathname) {
```

Comment:
this should import from tools instead of using a global.

Reply:
fixed, thanks